pkg/apis/infrastructure/v1alpha2/awsmachine_types.go

```diff
@@ -83,7 +83,8 @@ type AWSMachineSpec struct {
 // AWSMachineStatus defines the observed state of AWSMachine
 type AWSMachineStatus struct {
 	// Ready is true when the provider resource is ready.
-	Ready *bool `json:"ready,omitempty"`
+	// +optional
```

Comment:
I don't think you can call this optional unless it's a pointer - remove this?

Reply:
It seems controller tools allows it, if it’s not marked as optional, it’ll be marked as required in the CRD

Reply:
Does that mean it will error if it's not in the json/yaml when you POST it?

Reply:
Yes, the validation will fail, similar to what we've seen with ObjectMeta

Reply:
k, thanks.